Tavisca.Bootcamp.LanguageBasics.Exercise2/ForumPostEasy.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise2
+{
+    class ForumPostEasy
+    {
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            var currentTime = new List<TimeSpan>();
+            for (int i = 0; i < exactPostTime.Length; i++)
+            {
+                for (int j = i + 1; j < exactPostTime.Length; j++)
+                {
+                    if (exactPostTime[i] == exactPostTime[j])
+                    {
+                        if (showPostTime[i] != showPostTime[j])
+                            return "impossible";
+                    }
+                }
+                TimeSpan timeSpan = TimeSpan.Parse(exactPostTime[i]);
+                if (showPostTime[i].Contains("few"))
+                {
+                    TimeSpan time = new TimeSpan(timeSpan.Hours, timeSpan.Minutes, timeSpan.Seconds);
+                    currentTime.Add(time);
+                }
+                else if (showPostTime[i].Contains("minutes"))
+                {
```

Comment:
code duplication in if and else. Can we create a method for the same code here as well??